src/mesh/mesh_base.C

```diff
@@ -192,6 +192,17 @@ unsigned int MeshBase::get_elem_integer_index(const std::string & name) const
 
 
 
+bool MeshBase::has_elem_integer(const std::string & name) const
+{
+  for (auto i : index_range(_elem_integer_names))
```

Comment:
Why use indices here, rather than a range for over the container?  I wasn't planning on changing the container type but better safe than sorry.

Reply:
Only because I was copying the code in another function without much thinking. Changed to the range loop.